### PR TITLE
The deprecated null value for the method round has been changed to zero.

### PR DESCRIPTION
**Pull request by Parovozzzik**

### CODE REVIEW

**Review by fabpot — COMMENTED** [review]
Maybe we can move 554 as the first line of code instead?
Can you also add some tests (in Fixtures)?

**Comment by Parovozzzik** [comment]
@fabpot, I don't see what you mean about 554...  we can do it like this:

```
function twig_round($value, $precision = 0, $method = 'common')
{
    if (!isset($value)) {
        return 0;
    }
...
}
```

**Comment by fabpot** [comment]
Line is `$value = (float) $value;`. If you move this line to the beginning of the method, then, there is no other change to do as the value will always be a float.

File: tests/Fixtures/filters/round.test [hunk 1]
```diff
@@ -1,6 +1,7 @@
 --TEST--
 "round" filter
 --TEMPLATE--
+{{ null|round }}
```
Comment (fabpot) [inline tests/Fixtures/filters/round.test]: Can you also add two other tests for the `ceil` and `floor`  methods?

Comment (stof) [reply tests/Fixtures/filters/round.test]: `floor` is already tested (that's the last case), but `ceil`  is indeed missing.

Comment (Parovozzzik) [reply tests/Fixtures/filters/round.test]: Done

**Review by stof — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @Parovozzzik.

**Pull request merged**